crypto/build.gradle.kts

```diff
@@ -34,5 +35,84 @@ dependencies {
     testImplementation("org.junit.jupiter:junit-jupiter-api:$jupiterVersion")
     testImplementation("org.junit.jupiter:junit-jupiter-params:$jupiterVersion")
     testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:$jupiterVersion")
+}
+
+java{
+    withJavadocJar()
+    withSourcesJar()
+}
+
+tasks.javadoc {
+    if (JavaVersion.current().isJava9Compatible) {
+        (options as StandardJavadocDocletOptions).addBooleanOption("html5", true)
+    }
+}
+
+// Maven artifact
+val groupIdVal = "io.github.finschia"
+val artifactIdVal = "finschia-kt-crypto"
+val versionVal: String? = System.getProperty("VERSION")
+
+// Maven pom info
+val pomName = "finschia"
+val pomDesc = artifactIdVal
+val pomUrl = "https://github.com/Finschia/finschia-kt"
+val pomScmConnection = "scm:git:git://github.com/Finschia/finschia-kt.git"
+val pomDeveloperConnection = "scm:git:ssh://github.com/Finschia/finschia-kt.git"
+val pomScmUrl = "https://github.com/Finschia/finschia-kt"
+
+// Maven account
+val ossrhUserName = System.getenv("OSSRH_USERNAME")
+val ossrhPassword = System.getenv("OSSRH_PW")
```

Comment:
This configuration is the information to need in only our side. The other user to download this source does not need it. 
Is it no problem? And Isn't there any other way to use ci?